### DISCUSSION TITLE
Problem uploading a project with offline maps (*.mbtiles)

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#2459] [Q&A]
Hi, I am a forester and working with QField in the field ;-) OpenGis.ch gave me the opportunity to try out 'the cloud'. Thank you!

While uploading a project (350 MB in total) with QGis 3.16-3 Hannover and the latest experimental QFieldSync version v4.0.0-beta13 the upload is not completed (it freezes at 34%).
I assume it has something to do with 5 offline maps (*.mbtiles format, the largest with a maximum of 34 MB). Is this format supported?
The message log says it takes to long... When synchronizing afterwords, I got a error about aborted uploads.

![Schermafdruk_2021-09-17_17-44-12](https://user-images.githubusercontent.com/57868975/133837493-3bd65c73-79b0-4e46-bf6f-b36cd8ca4007.png)
![Schermafdruk_2021-09-17_20-32-42](https://user-images.githubusercontent.com/57868975/133837516-c29c5815-10f4-467d-82b2-8d8b8c169f77.png)
![Schermafdruk_2021-09-17_20-07-34](https://user-images.githubusercontent.com/57868975/133837524-c2b7849b-46ef-4ede-ab08-c0320704793d.png)
--

**Comment by nirvn** [comment]
Hello field worker using QField in the field. 

This should unlock mbtiles support: https://github.com/opengisch/libqfieldsync/pull/15

**Reply by BoswachterMarc** [reply]
Still no luck on this side ;-)
Message "duurt te lang" = " takes to long" 

![afbeelding](https://user-images.githubusercontent.com/57868975/133942958-54c70c53-3269-47eb-8840-90c84174100e.png)

**Reply by BoswachterMarc** [reply]
See also https://github.com/opengisch/qfieldsync/issues/322

![PrintscreenProbleem](https://user-images.githubusercontent.com/57868975/134052352-b816031e-18f9-46e3-9b40-565b7fec3324.PNG)

**Reply by BoswachterMarc** [reply]
I tried uploading a whole project once more and again I got the message of a network time-out. This time I worked on linux mint 20 Xfce with the experimental QFieldSync plug-in v4.0.0-beta15. I did not get the problem of using ASCII encoded filename.
Because of the time-out problem, **I did not upload/sync all the files at once (prefering local storage as 'single source of truth') but one for one. This workaround worked well and the whole project was finally synced and available in QField on my mobile.**
In other words, we are ready for the field work using QFieldCloud ;-)

![afbeelding](https://user-images.githubusercontent.com/57868975/136293036-b62ff954-614c-4760-991c-6adabc81df64.png)

**Reply by BoswachterMarc** [reply]
I uploaded a new project and invited 3 collaborators. Again, uploading all files at once didn't work but one for one worked fine.
Next I made some changes in the forest and synced afterwards. It all went flawless!

**Reply by BoswachterMarc** [reply]
At the office, I made some small changes to the project and tried syncing back (T23:19) from local (qgis on pc) to cloud version. That moment I had another time-out and not all the changes (only *.gpkg and *.qgs files!) were synced at once (freeze at 98%). See printscreens. After a second sync attempt, just 2 out of 3 files were still available for sync and with the second attempt the two remaining files were also synced.

![afbeelding](https://user-images.githubusercontent.com/57868975/139337801-07779d1c-8930-4e00-8507-b6cc21ae599c.png)
![afbeelding](https://user-images.githubusercontent.com/57868975/139337920-98bf8c03-e498-47ce-ac52-6ce7325e6b88.png)

**Comment by nirvn** [comment]
File-based vector tile dataset support needed a bit more love on the QGIS side of things (https://github.com/qgis/QGIS/pull/45238), you'll need upcoming QGIS 3.22 for it to be fully supported.

**Reply by mbernasocchi** [reply]
@BoswachterMarc do you by any chance have the possibility to try with the nightly builds of QGIS?

**Reply by BoswachterMarc** [reply]
I should look up what that means...

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
